src/Symfony/Component/Serializer/Normalizer/NormalizerInterface.php

```diff
@@ -25,7 +25,7 @@
     /**
      * Normalizes an object into a set of arrays/scalars.
      *
-     * @param object $object  Object to normalize
+     * @param mixed $object  Object to normalize
```

Comment:
Add a space after `mixed` to align argument names and satisfy Symfony Code Style rules.